packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -224,14 +224,14 @@ export class KeybindingWidget extends ReactWidget implements StatefulWidget {
                     if (chunkIndex !== 0) {
                         collection.push({ value: '+', match: false, key: false });
                     }
-                    const chunkMatches = queryItems.includes(matchChunks[chunkIndex].toLocaleLowerCase());
-                    match ||= chunkMatches;
+                    const indexOfTerm = searchTerms.indexOf(matchChunks[chunkIndex].toLocaleLowerCase());
+                    const chunkMatches = indexOfTerm > -1 ? !!searchTerms.splice(indexOfTerm, 1) : false;
                     collection.push({ value: chunk, match: chunkMatches, key: true });
                 });
                 return collection;
             }, []);
             item.labels.keybinding = { value: item.labels.keybinding.value, segments };
-            return match;
+            return !searchTerms.length;
```

Comment:
This now only returns true if _all_ chunks were matched (and so all removed from `searchTerms`, leaving none); previously it returned true if _any_ chunk matched. Is that deliberate?

Reply:
Yes
Part of the reason as to why searching for `Ctrl+U` returned more than the expected entries was because if only `Ctrl` matched then it would return that entry. Thus you would end with results for `Ctrl+K` and `Ctrl+T` when searching `Ctrl+U`.